src/utils.js

```diff
@@ -1215,3 +1215,41 @@ export function mergeDeep(target, ...sources) {
 
   return mergeDeep(target, ...sources);
 }
+
+/**
+ * returns a hash of a string using a fast algorithm
+ * source: https://stackoverflow.com/a/52171480/845390
+ * @param str
+ * @param seed (optional)
+ * @returns {string}
+ */
+export function simpleHash(str, seed = 0) {
+  // IE doesn't support imul
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/imul#Polyfill
+  if (!Math.imul) {
+    Math.imul = function(opA, opB) {
```

Comment:
I was under the impression that we were not allowing polyfills.

I could be wrong but I thought I read / heard that somewhere...